dynatrace-aws-logs.sh

```diff
@@ -158,7 +178,16 @@ arguments:
     TENANT_ID="" # NOT USED IN THIS CASE
   fi
 
+  if [[ "$USE_EXISTING_ACTIVE_GATE" == "false" ]] && ! [[ "${TARGET_URL}" =~ $DYNATRACE_TARGET_URL_REGEX ]]; then
+      echo "Invalid value for parameter --target-url. Example of valid url for deployment with ActiveGate: https://environment-id.live.dynatrace.com"
+      exit 1
+  elif [[ "$USE_EXISTING_ACTIVE_GATE" == "true" ]] && ! [[ "${TARGET_URL}" =~ $ACTIVE_GATE_TARGET_URL_REGEX ]]; then
+      echo "Invalid value for parameter --target-url. Example of valid url for deployment without ActiveGate: https://environment-active-gate-url:9999/e/environment-id"
+      exit 1
+  fi
+
   print_params_deploy
+  check_api_token
```

Comment:
I'd move the check above printing params - for me printing params is like saying checks are already ok

Reply:
Firstly I wanted to add checking api token before printing but in Azure solution as we have two modes there and we don't print params in interactive mode I needed to do checking right before deploying so in aws I did it in the same way to be consistent.
Anyone has the same feeling it should be before printing?